zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -37,6 +39,10 @@
 import zaza.openstack.utilities.openstack as zaza_openstack
 import zaza.openstack.utilities.generic as generic_utils
 
+# Disable warnings for ssl_verify=false
+urllib3.disable_warnings(
+    urllib3.exceptions.InsecureRequestWarning
```

Comment:
We have the CA in use when we make requests, can we not setup the urllib3 requests in such a way that they successfully verify with the test's CA?

Reply:
I guess we most certainly can. I was personally opinionated against going that extra mile because it's a test deployment and hence i simply suppressed those warnings. What's your take ? Should we ?